tests/run.sh

```diff
@@ -18,20 +27,24 @@ RESULT_DIR="results/$NAME/$TEST_DIR"
 mkdir -p $RESULT_DIR
 rm -rf $RESULT_DIR/*
 
+SECONDS=0
```

Comment:
Could you make a comment (maybe with a reference to the manual) about how this variable works?

Reply:
yeah, it's a bit obscure

Reply:
fixed